fluid/PaddleCV/human_pose_estimation/utils/transforms.py

```diff
@@ -0,0 +1,123 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) Microsoft
+# Licensed under the MIT License.
+# Written by Bin Xiao (Bin.Xiao@microsoft.com)
+# ------------------------------------------------------------------------------
```

Comment:
same as above.  How other framework handle the reference like this?

Reply:
Done.